{
	"id": "ALSA-2021:4356",
	"title": "Moderate: kernel security, bug fix, and enhancement update",
	"description": "The kernel packages contain the Linux kernel, the core of any Linux operating system.\n\nSecurity Fix(es):\n* kernel: out-of-bounds reads in pinctrl subsystem (CVE-2020-0427)\n* kernel: Improper input validation in some Intel(R) Ethernet E810 Adapter drivers (CVE-2020-24502)\n* kernel: Insufficient access control in some Intel(R) Ethernet E810 Adapter drivers (CVE-2020-24503)\n* kernel: Uncontrolled resource consumption in some Intel(R) Ethernet E810 Adapter drivers (CVE-2020-24504)\n* kernel: Fragmentation cache not cleared on reconnection (CVE-2020-24586)\n* kernel: Reassembling fragments encrypted under different keys (CVE-2020-24587)\n* kernel: wifi frame payload being parsed incorrectly as an L2 frame (CVE-2020-24588)\n* kernel: Forwarding EAPOL from unauthenticated wifi client (CVE-2020-26139)\n* kernel: accepting plaintext data frames in protected networks (CVE-2020-26140)\n* kernel: not verifying TKIP MIC of fragmented frames (CVE-2020-26141)\n* kernel: accepting fragmented plaintext frames in protected networks (CVE-2020-26143)\n* kernel: accepting unencrypted A-MSDU frames that start with RFC1042 header (CVE-2020-26144)\n* kernel: accepting plaintext broadcast fragments as full frames (CVE-2020-26145)\n* kernel: powerpc: RTAS calls can be used to compromise kernel integrity (CVE-2020-27777)\n* kernel: locking inconsistency in tty_io.c and tty_jobctrl.c can lead to a read-after-free (CVE-2020-29660)\n* kernel: buffer overflow in mwifiex_cmd_802_11_ad_hoc_start function via a long SSID value (CVE-2020-36158)\n* kernel: slab out-of-bounds read in hci_extended_inquiry_result_evt()  (CVE-2020-36386)\n* kernel: Improper access control in BlueZ may allow information disclosure vulnerability. (CVE-2021-0129)\n* kernel: Use-after-free in ndb_queue_rq() in drivers/block/nbd.c (CVE-2021-3348)\n* kernel: Linux kernel eBPF RINGBUF map oversized allocation (CVE-2021-3489)\n* kernel: double free in bluetooth subsystem when the HCI device initialization fails (CVE-2021-3564)\n* kernel: use-after-free in function hci_sock_bound_ioctl() (CVE-2021-3573)\n* kernel: eBPF 32-bit source register truncation on div/mod (CVE-2021-3600)\n* kernel: DoS in rb_per_cpu_empty() (CVE-2021-3679)\n* kernel: Mounting overlayfs inside an unprivileged user namespace can reveal files (CVE-2021-3732)\n* kernel: heap overflow in __cgroup_bpf_run_filter_getsockopt() (CVE-2021-20194)\n* kernel: Race condition in sctp_destroy_sock list_del (CVE-2021-23133)\n* kernel: fuse: stall on CPU can occur because a retry loop continually finds the same bad inode (CVE-2021-28950)\n* kernel: System crash in intel_pmu_drain_pebs_nhm in arch/x86/events/intel/ds.c (CVE-2021-28971)\n* kernel: protection can be bypassed to leak content of kernel memory (CVE-2021-29155)\n* kernel: improper input validation in tipc_nl_retrieve_key function in net/tipc/node.c (CVE-2021-29646)\n* kernel: lack a full memory barrier may lead to DoS (CVE-2021-29650)\n* kernel: local escalation of privileges in handling of eBPF programs (CVE-2021-31440)\n* kernel: protection of stack pointer against speculative pointer arithmetic can be bypassed to leak content of kernel memory (CVE-2021-31829)\n* kernel: out-of-bounds reads and writes due to enforcing incorrect limits for pointer arithmetic operations by BPF verifier (CVE-2021-33200)\n* kernel: reassembling encrypted fragments with non-consecutive packet numbers (CVE-2020-26146)\n* kernel: reassembling mixed encrypted/plaintext fragments (CVE-2020-26147)\n* kernel: the copy-on-write implementation can grant unintended write access because of a race condition in a THP mapcount check (CVE-2020-29368)\n* kernel: flowtable list del corruption with kernel BUG at lib/list_debug.c:50 (CVE-2021-3635)\n* kernel: NULL pointer dereference in llsec_key_alloc() in net/mac802154/llsec.c (CVE-2021-3659)\n* kernel: setsockopt System Call Untrusted Pointer Dereference Information Disclosure (CVE-2021-20239)\n* kernel: out of bounds array access in drivers/md/dm-ioctl.c (CVE-2021-31916)",
	"affected": [
		{
			"vulnerable": true,
			"package": {
				"ecosystem": "AlmaLinux:8",
				"name": "kernel-tools-libs-devel"
			},
			"ranges": [
				{
					"type": "ECOSYSTEM",
					"events": [
						{
							"introduced": "0"
						},
						{
							"fixed": "4.18.0-348.el8"
						}
					]
				}
			]
		}
	],
	"vulnerabilities": [
		{
			"cve": "CVE-2020-27777",
			"references": [
				{
					"name": "CVE-2020-27777",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-27777"
				}
			]
		},
		{
			"cve": "CVE-2020-36158",
			"references": [
				{
					"name": "CVE-2020-36158",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-36158"
				}
			]
		},
		{
			"cve": "CVE-2021-0129",
			"references": [
				{
					"name": "CVE-2021-0129",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-0129"
				}
			]
		},
		{
			"cve": "CVE-2021-28950",
			"references": [
				{
					"name": "CVE-2021-28950",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-28950"
				}
			]
		},
		{
			"cve": "CVE-2021-3659",
			"references": [
				{
					"name": "CVE-2021-3659",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-3659"
				}
			]
		},
		{
			"cve": "CVE-2021-3732",
			"references": [
				{
					"name": "CVE-2021-3732",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-3732"
				}
			]
		},
		{
			"cve": "CVE-2020-24503",
			"references": [
				{
					"name": "CVE-2020-24503",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-24503"
				}
			]
		},
		{
			"cve": "CVE-2020-36386",
			"references": [
				{
					"name": "CVE-2020-36386",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-36386"
				}
			]
		},
		{
			"cve": "CVE-2021-3564",
			"references": [
				{
					"name": "CVE-2021-3564",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-3564"
				}
			]
		},
		{
			"cve": "CVE-2020-29660",
			"references": [
				{
					"name": "CVE-2020-29660",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-29660"
				}
			]
		},
		{
			"cve": "CVE-2020-24588",
			"references": [
				{
					"name": "CVE-2020-24588",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-24588"
				}
			]
		},
		{
			"cve": "CVE-2021-28971",
			"references": [
				{
					"name": "CVE-2021-28971",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-28971"
				}
			]
		},
		{
			"cve": "CVE-2021-29646",
			"references": [
				{
					"name": "CVE-2021-29646",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-29646"
				}
			]
		},
		{
			"cve": "CVE-2021-31829",
			"references": [
				{
					"name": "CVE-2021-31829",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-31829"
				}
			]
		},
		{
			"cve": "CVE-2020-24586",
			"references": [
				{
					"name": "CVE-2020-24586",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-24586"
				}
			]
		},
		{
			"cve": "CVE-2020-24587",
			"references": [
				{
					"name": "CVE-2020-24587",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-24587"
				}
			]
		},
		{
			"cve": "CVE-2020-26141",
			"references": [
				{
					"name": "CVE-2020-26141",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-26141"
				}
			]
		},
		{
			"cve": "CVE-2020-26143",
			"references": [
				{
					"name": "CVE-2020-26143",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-26143"
				}
			]
		},
		{
			"cve": "CVE-2020-26145",
			"references": [
				{
					"name": "CVE-2020-26145",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-26145"
				}
			]
		},
		{
			"cve": "CVE-2021-20239",
			"references": [
				{
					"name": "CVE-2021-20239",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-20239"
				}
			]
		},
		{
			"cve": "CVE-2021-29650",
			"references": [
				{
					"name": "CVE-2021-29650",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-29650"
				}
			]
		},
		{
			"cve": "CVE-2021-33200",
			"references": [
				{
					"name": "CVE-2021-33200",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-33200"
				}
			]
		},
		{
			"cve": "CVE-2020-0427",
			"references": [
				{
					"name": "CVE-2020-0427",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-0427"
				}
			]
		},
		{
			"cve": "CVE-2021-3489",
			"references": [
				{
					"name": "CVE-2021-3489",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-3489"
				}
			]
		},
		{
			"cve": "CVE-2020-26147",
			"references": [
				{
					"name": "CVE-2020-26147",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-26147"
				}
			]
		},
		{
			"cve": "CVE-2020-29368",
			"references": [
				{
					"name": "CVE-2020-29368",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-29368"
				}
			]
		},
		{
			"cve": "CVE-2021-31916",
			"references": [
				{
					"name": "CVE-2021-31916",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-31916"
				}
			]
		},
		{
			"cve": "CVE-2020-26140",
			"references": [
				{
					"name": "CVE-2020-26140",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-26140"
				}
			]
		},
		{
			"cve": "CVE-2020-26144",
			"references": [
				{
					"name": "CVE-2020-26144",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-26144"
				}
			]
		},
		{
			"cve": "CVE-2020-26146",
			"references": [
				{
					"name": "CVE-2020-26146",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-26146"
				}
			]
		},
		{
			"cve": "CVE-2021-20194",
			"references": [
				{
					"name": "CVE-2021-20194",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-20194"
				}
			]
		},
		{
			"cve": "CVE-2021-23133",
			"references": [
				{
					"name": "CVE-2021-23133",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-23133"
				}
			]
		},
		{
			"cve": "CVE-2020-26139",
			"references": [
				{
					"name": "CVE-2020-26139",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-26139"
				}
			]
		},
		{
			"cve": "CVE-2021-31440",
			"references": [
				{
					"name": "CVE-2021-31440",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-31440"
				}
			]
		},
		{
			"cve": "CVE-2021-3348",
			"references": [
				{
					"name": "CVE-2021-3348",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-3348"
				}
			]
		},
		{
			"cve": "CVE-2021-3573",
			"references": [
				{
					"name": "CVE-2021-3573",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-3573"
				}
			]
		},
		{
			"cve": "CVE-2021-3600",
			"references": [
				{
					"name": "CVE-2021-3600",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-3600"
				}
			]
		},
		{
			"cve": "CVE-2021-3635",
			"references": [
				{
					"name": "CVE-2021-3635",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-3635"
				}
			]
		},
		{
			"cve": "CVE-2021-3679",
			"references": [
				{
					"name": "CVE-2021-3679",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-3679"
				}
			]
		},
		{
			"cve": "CVE-2021-29155",
			"references": [
				{
					"name": "CVE-2021-29155",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2021-29155"
				}
			]
		},
		{
			"cve": "CVE-2020-24504",
			"references": [
				{
					"name": "CVE-2020-24504",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-24504"
				}
			]
		},
		{
			"cve": "CVE-2020-24502",
			"references": [
				{
					"name": "CVE-2020-24502",
					"source": "errata.almalinux.org",
					"tags": [
						"cve"
					],
					"url": "https://vulners.com/cve/CVE-2020-24502"
				}
			]
		}
	],
	"references": [
		{
			"name": "CVE-2020-26144",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-26144"
		},
		{
			"name": "CVE-2021-31916",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-31916"
		},
		{
			"name": "CVE-2021-3600",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-3600"
		},
		{
			"name": "CVE-2020-24586",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-24586"
		},
		{
			"name": "CVE-2020-29660",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-29660"
		},
		{
			"name": "CVE-2020-36312",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-36312"
		},
		{
			"name": "CVE-2020-36386",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-36386"
		},
		{
			"name": "CVE-2021-29155",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-29155"
		},
		{
			"name": "CVE-2021-31829",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-31829"
		},
		{
			"name": "CVE-2021-33033",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-33033"
		},
		{
			"name": "CVE-2021-3564",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-3564"
		},
		{
			"name": "CVE-2020-26140",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-26140"
		},
		{
			"name": "CVE-2021-3659",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-3659"
		},
		{
			"name": "CVE-2020-26143",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-26143"
		},
		{
			"name": "CVE-2020-26146",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-26146"
		},
		{
			"name": "CVE-2020-27777",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-27777"
		},
		{
			"name": "CVE-2021-3348",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-3348"
		},
		{
			"name": "CVE-2021-3489",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-3489"
		},
		{
			"name": "CVE-2019-14615",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2019-14615"
		},
		{
			"name": "CVE-2020-26147",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-26147"
		},
		{
			"name": "CVE-2021-28971",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-28971"
		},
		{
			"name": "CVE-2020-26141",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-26141"
		},
		{
			"name": "CVE-2020-24588",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-24588"
		},
		{
			"name": "CVE-2020-29368",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-29368"
		},
		{
			"name": "CVE-2021-0129",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-0129"
		},
		{
			"name": "CVE-2021-20194",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-20194"
		},
		{
			"name": "CVE-2021-28950",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-28950"
		},
		{
			"name": "CVE-2021-29646",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-29646"
		},
		{
			"name": "CVE-2021-3679",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-3679"
		},
		{
			"name": "CVE-2020-24587",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-24587"
		},
		{
			"name": "CVE-2020-24504",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-24504"
		},
		{
			"name": "CVE-2021-3573",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-3573"
		},
		{
			"name": "CVE-2021-3635",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-3635"
		},
		{
			"name": "CVE-2020-24502",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-24502"
		},
		{
			"name": "CVE-2020-24503",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-24503"
		},
		{
			"name": "CVE-2020-26139",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-26139"
		},
		{
			"name": "CVE-2020-0427",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-0427"
		},
		{
			"name": "CVE-2020-36158",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-36158"
		},
		{
			"name": "CVE-2021-20239",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-20239"
		},
		{
			"name": "CVE-2021-23133",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-23133"
		},
		{
			"name": "CVE-2021-29650",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-29650"
		},
		{
			"name": "CVE-2021-31440",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-31440"
		},
		{
			"name": "CVE-2021-33200",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-33200"
		},
		{
			"name": "CVE-2021-3732",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2021-3732"
		},
		{
			"name": "CVE-2020-26145",
			"source": "errata.almalinux.org",
			"tags": [
				"cve"
			],
			"url": "https://vulners.com/cve/CVE-2020-26145"
		}
	],
	"published": "2021-11-09T09:08:02Z",
	"modified": "2021-11-12T10:21:01Z",
	"data_source": "alma-osv"
}
